### PR TITLE
Fix container scenario to accept only signal number (#350)

### PR DESCRIPTION
**Pull request by jtydlack**

### CODE REVIEW

File: kraken/pod_scenarios/setup.py [hunk 1]
```diff
@@ -127,14 +129,20 @@ def container_run(kubeconfig_path,
...
-
 def container_killing_in_pod(cont_scenario, kubecli: KrknKubernetes):
     scenario_name = cont_scenario.get("name", "")
     namespace = cont_scenario.get("namespace", "*")
     label_selector = cont_scenario.get("label_selector", None)
     pod_names = cont_scenario.get("pod_names", [])
     container_name = cont_scenario.get("container_name", "")
-    kill_action = cont_scenario.get("action", "kill 1")
+    kill_action = cont_scenario.get("action", 1)
+    if not isinstance(kill_action, int):
```
Comment (paigerube14) [inline kraken/pod_scenarios/setup.py:139]: I dont think this if statement will ever be hit since we are setting a default if kill_action is not set

Comment (chaitanyaenr) [reply kraken/pod_scenarios/setup.py:139]: Think this checks if kill_action is an int if set, if not set in the config we go with 1. Thoughts?

File: kraken/pod_scenarios/setup.py [hunk 2]
```diff
@@ -127,14 +129,20 @@ def container_run(kubeconfig_path,
...
+        logging.error("Please make sure you inserted kill action number")
```
Comment (paigerube14) [inline kraken/pod_scenarios/setup.py:140]: can you maybe clarify in the comment here to end with "as a number" or "as an int"

Comment (chaitanyaenr) [inline kraken/pod_scenarios/setup.py:140]: Maybe change this to "Please make sure the action parameter in the config is an integer". Thoughts?

File: scenarios/openshift/container_etcd.yml [hunk 3]
```diff
@@ -3,6 +3,6 @@ scenarios:
   namespace: "openshift-etcd"
   label_selector: "k8s-app=etcd"
   container_name: "etcd"
-  action: "kill 1"
+  action: 1
```
Comment (chaitanyaenr) [inline scenarios/openshift/container_etcd.yml:6]: We need to update https://github.com/redhat-chaos/krkn/blob/main/docs/container_scenarios.md#example-config and https://github.com/redhat-chaos/krkn-hub/blob/main/container-scenarios/env.sh#L9 post this PR merge.

File: docs/container_scenarios.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ scenarios:
   pod_names:  # This is optional, can take out and will select all pods with given namespace and label
   - <pod_name>
   count: <number of containers to disrupt, default=1>
-  action: <Action to run. For example kill 1 ( hang up ) or kill 9. Default is set to kill 1>
+  action: <number of kill action to run. For example 1 ( hang up ) or 9. Default is set to 1>
```
Comment (chaitanyaenr) [inline docs/container_scenarios.md:18]: s/number/signal

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

**Pull request merged**